### PR TITLE
Added the feature section in main website

### LINKED ISSUE #50: make a feature section in main website
**Is your feature request related to a problem? Please describe.**
make a component ```SystemFeature.js```  inside ```hotelontouch``` and provide a proper svg from undraw and write one heading ```Completely Open Sourced Hotel Management System```` and under that add the project description by copying from Readme.

### PR DESCRIPTION
**Pull request by abhishek7997**
Fixes #50 

SystemFeature.js as been created in components folder, a new svg file is added in assets folder, a new section has been added in the homepage.

Changes done
- [Added a new file SystemFeature.js in hotelontouch\src\components]
- [Added <SystemFeature/> component in HomePage.js below <AllContributors /> component]

Screenshots of the changes (If any) -
image

Note: Please check Allow edits from maintainers. if you would like us to assist in the PR.

#### ✅️ By submitting this PR, I have verified the following

- [x] Worked on forked repo and starred the main repo
- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [ ] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

**Comment by abhishek7997** [comment]
@ayan-biswas0412 
Please elaborate on what exactly should I change.

**Comment by ayan-biswas0412** [comment]
> @ayan-biswas0412
> Please elaborate on what exactly should I change.

please modify our previously added welcome card not add any other card , all you have done please append that in welcome card and contribute to our project button should be there

**Comment by abhishek7997** [comment]
> > @ayan-biswas0412
> > Please elaborate on what exactly should I change.
> 
> please modify our previously added welcome card not add any other card , all you have done please append that in welcome card and contribute to our project button should be there

There is no welcome card, at least I cannot find one. Is it the Jumbotron that is displayed on the top of the page?

**Comment by abhishek7997** [comment]
image

@ayan-biswas0412 
Is this correct?

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]
enhance the button add a github logo to that

**Comment by ayan-biswas0412** [comment]
@JanviMahajan14 review pls

**Comment by abhishek7997** [comment]
> enhance the button add a github logo to that

Will be doing that right now

**Comment by abhishek7997** [comment]
@ayan-biswas0412 
Sir, please check the new PR, I have added the github icon as well.

File: hotelontouch/src/assets/undraw_project_team_lc5a.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...309H594.21808a3.00328,3.00328,0,0,1-3-3v-1.17237a3.00328,3.00328,0,0,1,3-3h20.92334a3.00329,3.00329,0,0,1,3,3v1.17237A3.00329,3.00329,0,0,1,615.14142,410.79309Z" transform="translate(-206.46668 -235.93615)" fill="#fff"/><path d="M615.14142,425.60656H594.21808a3.00328,3.00328,0,0,1-3-3V421.4342a3.00328,3.00328,0,0,1,3-3h20.92334a3.00328,3.00328,0,0,1,3,3v1.17236A3.00328,3.00328,0,0,1,615.14142,425.60656Z" transform="translate(-206.46668 -235.93615)" fill="#fff"/><path d="M481.462,320.19494H460.53839a3.00328,3.00328,0,0,1-3-3v-1.17236a3.00328,3.00328,0,0,1,3-3H481.462a3.00328,3.00328,0,0,1,3,3v1.17236A3.00328,3.00328,0,0,1,481.462,320.19494Z" transform="translate(-206.46668 -235.93615)" fill="#fff"/><path d="M553.639,257.83117H532.71564a3.00328,3.00328,0,0,1-3-3v-1.17236a3.00328,3.00328,0,0,1,3-3H553.639a3.00328,3.00328,0,0,1,3,3v1.17236A3.00328,3.00328,0,0,1,553.639,257.83117Z" transform="translate(-206.46668 -235.93615)" fill="#fff"/><path d="M649.11285,310.86486H628.18927a3.00328,3.00328,0,0,1-3-3V306.6925a3.00328,3.00328,0,0,1,3-3h20.92358a3.00328,3.00328,0,0,1,3,3v1.17236A3.00328,3.00328,0,0,1,649.11285,310.86486Z" transform="translate(-206.46668 -235.93615)" fill="#fff"/></svg>
```
Comment (ayan-biswas0412) [inline hotelontouch/src/assets/undraw_project_team_lc5a.svg:1]: please rename the svg as welcome_iamge.svg and make necessary changes according to that , good work

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]

File: hotelontouch/src/assets/undraw_project_team_lc5a.svg (hunk 1 above)
Comment (abhishek7997) [reply hotelontouch/src/assets/undraw_project_team_lc5a.svg]: @ayan-biswas0412 sir, I have made the changes. Please check once again.

**Review by ayan-biswas0412 — APPROVED** [review]
nice work

**Comment by abhishek7997** [comment]
Thank you sir.

**Pull request merged**